tools/docker/image-pull.sh

```diff
@@ -33,15 +34,16 @@ pull_and_tag() {
 }
 
 main() {
-    OPTS=`getopt -o 'hvb:t:' --long verbose,help,base-image,tag -n 'parse-options' -- "$@"`
-
-    if [ $? != 0 ] ; then err "Failed parsing options." >&2 ; usage; exit 1 ; fi
+    if ! OPTS=$(getopt -o 'hb:t:' --long help,base-image,tag -n 'parse-options' -- "$@"); then
+        err "Failed parsing options." >&2
+        usage
+        exit 1
+    fi
 
     eval set -- "$OPTS"
 
     while true; do
         case "$1" in
-            -v | --verbose)         VERBOSE=true; shift ;;
```

Comment:
The removal of the verbose option is not mentioned in the commit message. It's probably OK to do it (I don't think it has any effect anyway), but in general we prefer such changes to be in a separate commit.

Don't bother doing it now though, just for future.